### PR TITLE
docs/vmanomaly - release 1.13.0 preparation

### PR DESCRIPTION
**Pull request by fred-navruzov**
### Describe Your Changes

[vmanomaly docs](https://docs.victoriametrics.com/anomaly-detection/) update for changes, introduced in v1.13.0

### Checklist

The following checks are **mandatory**:

- [x] My change adheres [VictoriaMetrics contributing guidelines](https://docs.victoriametrics.com/contributing/).

### CODE REVIEW

File: docs/anomaly-detection/components/scheduler.md [hunk 1]
```diff
@@ -374,20 +384,24 @@ In `BacktestingScheduler`, the inference window is *implicitly* defined as a per
 
 ### ISO format scheduler config example
 ```yaml
-scheduler:
- class: "scheduler.backtesting.BacktestingScheduler"
- from_iso: '2021-01-01T00:00:00Z'
- to_iso: '2021-01-14T00:00:00Z'
- fit_window: 'P14D'
- fit_every: 'PT1H'
+schedulers:
```
Comment (Cambaza) [inline docs/anomaly-detection/components/scheduler.md:387]: please, add n_jobs to config example and as a separate parameter in description

File: docs/anomaly-detection/CHANGELOG.md [hunk 2]
```diff
@@ -15,8 +15,19 @@ aliases:
...
+
+## v1.13.0
+Released: 2024-06-07
+- FEATURE: Introduced `min_dev_from_expected` [model common arg](/anomaly-detection/components/models/#minimal-deviation-from-expected), aimed at **reducing [false positives](https://victoriametrics.com/blog/victoriametrics-anomaly-detection-handbook-chapter-1/#false-positive)** in scenarios where deviations between the real value `y` and the expected value `yhat` are **relatively** high and may cause models to generate high [anomaly scores](https://docs.victoriametrics.com/anomaly-detection/faq/#what-is-anomaly-score). However, these deviations are not significant enough in **absolute values** to be considered anomalies based on domain knowledge. This parameter ensures that anomaly scores for data points where `|y - yhat| < min_dev_from_expected` are explicitly set to 0. If not set, the default behavior is `min_dev_from_expected=0` to maintain backward compatibility.
```
Comment (Cambaza) [inline docs/anomaly-detection/CHANGELOG.md:22]: Should all this info be in the changelog? maybe just `introduced parameter X. backward in/compatible + Link to docs` ?

**Review by Cambaza — CHANGES_REQUESTED** [review]

File: docs/anomaly-detection/CHANGELOG.md (hunk 2 above)
Comment (fred-navruzov) [reply docs/anomaly-detection/CHANGELOG.md:22]: I'd rather keep it here, however, shortened descriptions a little bit :+1:

File: docs/anomaly-detection/FAQ.md [hunk 3]
```diff
@@ -116,6 +116,37 @@ Configuration above will produce N intervals of full length (`fit_window`=14d +
...
+    image: victoriametrics/vmanomaly:latest
+    # ...
+    ports:
+      - "8490:8490"
+    restart: always
+    volumes:
+      - ./vmanomaly_config.yml:/config.yaml
+      - ./vmanomaly_license:/license
+      # map the host directory to the container directory
+      - vmanomaly_model_dump_dir:/vmanomaly/tmp/models
```
[the remark this answers is not in the document]
Comment (fred-navruzov) [reply docs/anomaly-detection/FAQ.md:136]: no, it shouldn't, as it's a volume, defined in outermost `volumes` section, see other rows , like
```yaml
victoriametrics:
    container_name: victoriametrics
    i# ...
    volumes:
      - vmdata-guide-vmanomaly-vmalert:/storage
```

File: docs/anomaly-detection/Overview.md [hunk 4]
```diff
@@ -167,7 +167,7 @@ schedulers:
 
 models:
   prophet:  # or use a model alias of your choice here
-    class: "model.prophet.ProphetModel"
+    class: "model.prophet.ProphetModel"  # or 'prophet' starting from v1.13.0
```
[the remark this answers is not in the document]
Comment (fred-navruzov) [reply docs/anomaly-detection/Overview.md]: updated throughout main places (yaml configs examples)

**Review by Cambaza — APPROVED** [review]

**Pull request merged**